collection-scripts/gather_sriov

```diff
@@ -47,19 +47,19 @@ function gather_ethtool(){
   ETHTOOL_LOG_PATH="${CONFIG_DAEMON_POD_LOG_PATH}/ethtool"
 
   # Get ip -o link show output.
-  OUT="$(oc exec -n openshift-sriov-network-operator ${1} -c sriov-network-config-daemon -- chroot /host \
+  OUT="$(oc exec -n ${SRIOV_NS} ${1} -c sriov-network-config-daemon -- chroot /host \
   /bin/bash -c "ip -o link show 2>/dev/null")"
 
   # Cut long interfaces names.
-  INTERFACES="$(echo "$OUT" | awk -F': ' '{print $2}' | cut -d '@' -f 1)"
+  INTERFACES="$(echo "$OUT" | awk -F': ' '{print $2}')"
 
   # Run ethtool for each interface except for lo.
   while IFS= read -r interface; do
-    if [ -z "$interface" ] || [ "$interface" = "lo" ]; then
+    if [ -z "$interface" ] || [ "$interface" = "lo" ] || [[ "$interface" == *"@"* ]]; then
```

Comment:
why?

Reply:
we want to remove all the veth (ovn-k) related interfaces.

there is no need for sriov must-gather to get that information.